tasks/webfont.js

```diff
@@ -389,38 +392,133 @@ module.exports = function(grunt) {
 			}
 		}
 
-		/**
-		 * Generate HTML demo page
-		 *
-		 * @param {Function} done
+		/*
+		 * Prepares context for compilation of template
 		 */
-		function generateDemoHtml(done) {
-			if (!o.htmlDemo) {
-				done();
-				return;
+		function prepareTemplateContext(key) {
+
+			var context = _.extend({}, o);
+
+			if (_.isUndefined(key)) {
+				return context;
 			}
 
-			// HTML should not contain relative paths
-			// If some styles was not included in CSS we should include them in HTML to properly render icons
+			if (key === 'html') {
+				return prepareHTMLContext(context);
+			}
+
+			logger.warn('Key specified to function `prepareTemplateContext` does not correspond to a particular method.');
```

Comment:
It could be just `throw` because it’s not a user error. And what does `key` means? As I understand it’s more like a type of a template output.